### PR TITLE
remove extra get on crd during imagestream processing (a conflict win…

### PR DESCRIPTION
**Pull request by gabemontero**
…dow reduction get)

### CODE REVIEW

File: pkg/stub/imagestreams.go [hunk 1]
```diff
@@ -134,11 +134,6 @@ func (h *Handler) processImageStreamWatchEvent(is *imagev1.ImageStream, deleted
 		h.crdwrapper.UpdateStatus(cfg)
 		return err
 	}
-	// refetch cfg to narrow conflict window
```
Comment (coreydaley) [inline pkg/stub/imagestreams.go]: Would a "RetryOnConflict" work here?

**Comment by gabemontero** [comment]
On Thu, Apr 25, 2019 at 9:23 PM Corey Daley <notifications@github.com>
wrote:

> *@coreydaley* commented on this pull request.
> ------------------------------
>
> In pkg/stub/imagestreams.go
> <https://github.com/openshift/cluster-samples-operator/pull/138#discussion_r278783256>
> :
>
> > @@ -134,11 +134,6 @@ func (h *Handler) processImageStreamWatchEvent(is *imagev1.ImageStream, deleted
>  		h.crdwrapper.UpdateStatus(cfg)
>  		return err
>  	}
> -	// refetch cfg to narrow conflict window
>
> Would a "RetryOnConflict" work here?
>
Don’t think so cause of multiple hits against the same field (removing from
a list of streams)

But the reality is the caching above rendered this less if not un necessary
- but let’s see how the testing goes

**Comment by gabemontero** [comment]
`fail [github.com/openshift/origin/test/extended/util/cli.go:739]: Apr 25 22:20:26.961: Unauthorized` on image eco

/retest

**Comment by gabemontero** [comment]
`fail [github.com/openshift/origin/test/extended/util/cli.go:739]: Apr 26 11:47:07.468: Unauthorized`

/retest

**Comment by gabemontero** [comment]
`fail [github.com/openshift/origin/test/extended/util/cli.go:739]: Apr 26 12:39:23.954: Unauthorized`

/retest

**Pull request merged**